remix-analyzer/test/analysis/test-contracts/solidity-v0.5/ballot.sol

```diff
@@ -141,5 +141,7 @@ contract Ballot {
     {
         winnerName = proposals[winningProposal()].name;
     }
+
+    function testWithArray (bytes32[] memory param) public {}
```

Comment:
should we have another test with `testWithStruct` also?

Reply:
0.5 version is not supporting ABIENCODERV2. We can have probably with 0.6.0 additions.

Reply:
ok , even with the `pragma` ?

Reply:
We was not getting that warning for 0.5.0 with ABIENCODERV2 pragma too, even on remix IDE